docs/data/install-xroot-client.md

```diff
@@ -1,97 +1,196 @@
+Installing and Using the XRootD Client 
+==============================================
```

Comment:
Since this doc is more than using the XRootD client, we should rename it to "Using XRootD"